presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformQuantifiedComparisonApplyToScalarApply.java

```diff
@@ -261,5 +250,17 @@ private static boolean shouldCompareValueWithLowerBound(QuantifiedComparisonExpr
             }
             throw new IllegalArgumentException("Unexpected quantifier: " + quantifiedComparison.getQuantifier());
         }
+
+        private ProjectNode projectExpressions(PlanNode input, Assignments subqueryAssignments)
+        {
+            Assignments assignments = Assignments.builder()
+                    .putIdentities(input.getOutputSymbols())
+                    .putAll(subqueryAssignments)
```

Comment:
I don't think you can put `subqueryAssignments` into project assignments. `subqueryAssignments` can have expression like `EXISTS` or `IN` which project does not support